migrations/sql/V2019.03.25.14.40__add_variance_table.sql

```diff
@@ -0,0 +1,33 @@
+CREATE TABLE IF NOT EXISTS variance
+(
+    variance_id           integer                PRIMARY KEY                           ,
+    compliance_article_id integer                                              NOT NULL,
+    mine_guid             uuid                                                 NOT NULL,
+    note                  character varying(300) DEFAULT ''::character varying NOT NULL,
+    issue_date            date                   DEFAULT '9999-12-31'::date    NOT NULL,
+    received_date         date                   DEFAULT '9999-12-31'::date    NOT NULL,
+    expiry_date           date                   DEFAULT '9999-12-31'::date    NOT NULL
+);
```

Comment:
We probably should have the 4 audit columns on this table too.

Reply:
Added 👍